cmd/root/root.go

```diff
@@ -105,7 +105,7 @@ func init() {
 			name:   config.FormatFlag,
 			short:  "f",
 			value:  "",
-			usage:  "Output format. [env-var|aws-credentials]",
+			usage:  "Output format. [env-var|aws-credentials|credentials-provider]",
```

Comment:
Should this be `credential-provider` or is `credentials-provider` fine?

Reply:
It should be `credential-provider` as per the [AWS documentation](https://docs.aws.amazon.com/sdkref/latest/guide/feature-process-credentials.html). Thanks for catching this.
```suggestion
			usage:  "Output format. [env-var|aws-credentials|credential-provider]",
```